vcr/vcr.go

```diff
@@ -153,3 +255,59 @@ func (c *vcr) convert(query concept.Query) map[string]leia.Query {
 
 	return qs
 }
+
+func (c *vcr) resolveAssertionKey(id did.DID) (did.URI, error) {
```

Comment:
I wrote sort of the same function in `auth/services/util.go:14`. Maybe we can move it to a common package.

Reply:
refactored it, parts to the VDR and endpoint resolvement to did.Document